### PR TITLE
dotCMS/core#21276 Suggestions menu with many options is broken

### PR DESCRIPTION
**Pull request by rjvelazco**

### CODE REVIEW

File: libs/block-editor/src/lib/extensions/components/suggestions-list-item/suggestions-list-item.component.ts [hunk 1]
```diff
@@ -37,4 +37,8 @@ export class SuggestionsListItemComponent implements FocusableOption, OnInit {
     unfocus() {
         this.element.nativeElement.style = '';
     }
+
+    scrollIntoView() {
+        this.element.nativeElement.scrollIntoViewIfNeeded(false);
```
Comment (fmontes) [inline libs/block-editor/src/lib/extensions/components/suggestions-list-item/suggestions-list-item.component.ts:42]: We need to support FF: https://caniuse.com/?search=scrollIntoViewIfNeeded

File: libs/block-editor/src/lib/extensions/components/suggestions-list-item/suggestions-list-item.component.scss [hunk 2]
```diff
@@ -1,10 +1,15 @@
 :host {
+    box-sizing: border-box;
     display: flex;
     gap: 1rem;
     padding: 0.25rem 0.5rem;
     background-color: #fff;
     cursor: pointer;
     align-items: center;
+    height: 40px;
```
Comment (fmontes) [inline libs/block-editor/src/lib/extensions/components/suggestions-list-item/suggestions-list-item.component.scss:9]: Why do we need this?

File: libs/block-editor/src/lib/extensions/components/suggestions-list-item/suggestions-list-item.component.scss [hunk 3]
```diff
@@ -1,10 +1,15 @@
...
+    &:hover {
+        background: #eee;
```
Comment (fmontes) [inline libs/block-editor/src/lib/extensions/components/suggestions-list-item/suggestions-list-item.component.scss:11]: var?

**Review by fmontes — CHANGES_REQUESTED** [review]

**Review by alfredo-dotcms — DISMISSED** [review]

File: libs/block-editor/src/lib/extensions/components/suggestions-list-item/suggestions-list-item.component.ts (hunk 1 above)
Comment (rjvelazco) [reply libs/block-editor/src/lib/extensions/components/suggestions-list-item/suggestions-list-item.component.ts:42]: I changed `scrollIntoViewIfNeeded` for `scrollIntoView` which is supported in FF (https://caniuse.com/?search=scrollIntoView)

File: libs/block-editor/src/lib/extensions/components/suggestions-list-item/suggestions-list-item.component.scss (hunk 2 above)
Comment (rjvelazco) [reply libs/block-editor/src/lib/extensions/components/suggestions-list-item/suggestions-list-item.component.scss:9]: Suggestions-list-item had a non-explicit `height` of `39.Xpx`, I rounded it to 40px per list item to set a `max-height` of `400px` on the suggestion-list container to be able to display up to 10 options and more than that must be scroll.

**Pull request closed without merging**